setup.py

```diff
@@ -25,15 +25,15 @@
     python_requires='>=3.8',
     install_requires=[
         "abnf~=2.2.0",
-        "chardet~=5.0.0",
+        "chardet>=5.0.0",
         "cint>=1.0.0",
         "fickling>=0.0.8",
         "graphviz>=0.20.1",
         "intervaltree>=2.4.0",
         "jinja2>=2.1.0",
         "kaitaistruct~=0.10",
         "networkx>=2.6.3",
-        "pdfminer.six==20220524",
+        "pdfminer.six>=20220524",
```

Comment:
I tested PolyFile against all of the versions in this range, so they should be good. PolyFile is tightly coupled with `pdfminer.six`, though, so a future update to `pdfminer.six` could very easily break this integration, so we unfortunately can't leave an open-ended version range.
```suggestion
        "pdfminer.six>=20220524,<=20240706",
```